components/links.tsx

```diff
@@ -120,12 +122,10 @@ export function memberLink(member: MemberContent) {
   return <External href={memberURL(member)}>{member.Name}</External>
 }
 
-export const twitterShareLink = (publication: Testimony) => {
+export const twitterShareLink = (publication: Testimony, t: TFunction) => {
```

Comment:
this is only being used in `components/publish/panel/YourTestimony.tsx`, I used the `useTranslation` hook in that file and passed it in here instead